src/components/Button/Button.test.tsx

```diff
@@ -50,7 +50,7 @@ describe('Button', () => {
   })
 
   describe('with extra className and disabled', () => {
-    const  { container } = render(<Button className={this.className}>Найти</Button>)
+    const  { container } = render(<Button className={this.className} disabled={true}>Найти</Button>)
```

Comment:
Да. Если что, для булевых можно короче:
```suggestion
    const  { container } = render(<Button className={this.className} disabled>Найти</Button>)
```

Но оставь, как есть.